### PR TITLE
HTML movie seat booking.

### PR DESCRIPTION
**Pull request by al0tak**
[Demo](https://yaripey.github.io/cinema-html-css/)

### CODE REVIEW

File: submissions/yaripey/html-movie-seat-booking/index.html [hunk 1]
```diff
@@ -0,0 +1,485 @@
...
+          </ul>
+        </nav>
+      </div>
+    </header>
+
+    <nav class="site-path">
+      <ol>
+        <li><a class="path-link" href="#">Фільми</a></li>
+        <li><a class="path-link" href="#">Сонік У Кіно</a></li>
+        <li><a class="path-link active" href="#">Придбати білет</a></li>
```
Comment (A-Ostrovnyy) [inline submissions/yaripey/html-movie-seat-booking/index.html]: In Ukrainian "квиток". You use that word later)

File: submissions/yaripey/html-movie-seat-booking/index.html [hunk 2]
```diff
@@ -0,0 +1,485 @@
...
+    <header>
+      <div class="header-left">
+        <a class="logo" href="#">Camera Roll</a>
+      </div>
+      <div class="header-right">
+        <nav class="settings-buttons">
+          <ul>
+            <li><a class="settings-buttons-link settings" href="#"></a></li>
+            <li><a class="settings-buttons-link profile" href="#"></a></li>
+            <li><a class="settings-buttons-link sign-out" href="#"></a></li>
```
Comment (A-Ostrovnyy) [inline submissions/yaripey/html-movie-seat-booking/index.html:25]: Logically it's no navigation, no need to use navigation markup.

File: submissions/yaripey/html-movie-seat-booking/index.html [hunk 3]
```diff
@@ -0,0 +1,485 @@
...
+                  <div class="input-label">18 лют</div>
+                </label>
+                <label>
+                  <input
+                    checked
+                    class="radio-box"
+                    type="radio"
+                    name="date"
+                    id="date3"
+                  />
```
Comment (A-Ostrovnyy) [inline submissions/yaripey/html-movie-seat-booking/index.html]: Add attribute `value` to the input.

File: submissions/yaripey/html-movie-seat-booking/index.html [hunk 4]
```diff
@@ -0,0 +1,485 @@
...
+                  <div class="input-label">19 лют</div>
```
Comment (A-Ostrovnyy) [inline submissions/yaripey/html-movie-seat-booking/index.html]: For date use `time` tag with `datetime` attribute.

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
Good work!
Extra like for the Ukrainian language.
Remove files not for review(images). Also, don't add to commit edited config files.
See my comments below.

**Comment by ses3332002** [comment]
Why you used lang="en" in html tag? 
I guess for Ukrainian it must be "uk".

**Comment by al0tak** [comment]
> Why you used lang="en" in html tag?
> I guess for Ukrainian it must be "uk".

Thanks for pointing that out!

**Review by A-Ostrovnyy — APPROVED** [review]
Well done!

**Pull request merged**